internal/remoteconfig/remoteconfig.go

```diff
@@ -53,8 +53,20 @@ const (
 	ASMUserBlocking
 	// ASMCustomRules represents the capability for ASM to receive and use user-defined security rules
 	ASMCustomRules
-	// ASMCustomRules represents the capability for ASM to receive and use user-defined blocking responses
+	// ASMCustomBlockingResponse represents the capability for ASM to receive and use user-defined blocking responses
 	ASMCustomBlockingResponse
+	// ASMTrustedIPs represents Trusted IPs through the ASM product
+	ASMTrustedIPs
+	// ASMApiSecuritySampleRate represents API Security sampling rate
+	ASMApiSecuritySampleRate
+	// APMTracingSampleRate represents the rate at which to sample traces from APM client libraries
+	APMTracingSampleRate
+	// APMTracingLogsInjection enables APM client libraries to inject trace ids into log records
+	APMTracingLogsInjection
+	// APMTracingHTTPHeaderTags enables APM client libraries to tag http header values to http server or client spans
+	APMTracingHTTPHeaderTags
+	// APMTracingCustomTags enables APM client to set custom tags on all spans
+	APMTracingCustomTags
```

Comment:
The order of these capabilities matters that's why I had to add a few that aren't currently used. Each capability is defined by its rank in the list.
The full and detailed documentation for RC capabilities is internal, I can share it offline if the reviewer is interested.

Reply:
IIUC, our tracer only supports APMTracingSampleRate and APMTracingHTTPHeaderTags, but no APMTracingLogsInjection and APMTracingCustomTags. From the JIRA issue, it's not clear what is the support of the latter. Probably I'm missing context.

Reply:
APMTracingLogsInjection won't be supported since there is no automatic logs injection in Go. APMTracingCustomTags will be supported in future versions though.